Makefile

```diff
@@ -126,41 +118,49 @@ endif
 
 all: setup run deploy_nodes_with_install
 
-destroy: destroy_nodes delete_minikube kill_port_forwardings destroy_onprem stop_load_balancer
+destroy: destroy_nodes delete_minikube delete_kind kill_port_forwardings destroy_onprem stop_load_balancer
 
 ###############
 # Environment #
 ###############
 setup:
 	./scripts/create_full_environment.sh
 
-create_environment: image_build bring_assisted_service start_minikube
+create_environment: image_build bring_assisted_service create_hub_cluster
 
 image_build:
 	scripts/pull_dockerfile_images.sh
 	sed 's/^FROM .*assisted-service.*:latest/FROM $(subst /,\/,${SERVICE})/' Dockerfile.assisted-test-infra | \
 		$(CONTAINER_COMMAND) build --network=host -t $(IMAGE_NAME) -f- .
 	$(CONTAINER_COMMAND) tag $(IMAGE_NAME) test-infra:latest  # For backwards computability
 
+create_hub_cluster:
```

Comment:
should probably added in the `.PHONY` (and for `delete_hub_cluster`,  `delete_kind` as well)